graphql/schema/types/distro.graphql

```diff
@@ -1,10 +1,142 @@
+###### ENUMS ######
 enum DistroSettingsAccess {
   ADMIN
   CREATE
   EDIT
   VIEW
 }
 
+enum DistroSettingsSection {
+  GENERAL
+  PROVIDER
+  TASK
+  HOST
+  PROJECT
+}
+
+enum DistroOnSaveOperation {
+  DECOMMISSION
+  RESTART_JASPER
+  REPROVISION
+  NONE
+}
+
+###### INPUTS ######
+input SaveDistroOpts {
```

Comment:
Not sure if this is a practice worth keeping around, but we do usually name inputs with `Input` at the end
```suggestion
input SaveDistroInput {
```

Reply:
oops sorry yes!